test/onnx/test_onnx_opset.py

```diff
@@ -106,7 +106,7 @@ def test_maxpool(self):
                     {"name": "kernel_shape", "ints": [2], "type": 7},
                     {"name": "pads", "ints": [0, 0], "type": 7},
                     {"name": "strides", "ints": [1], "type": 7}]}]
-        ops = {9 : ops_9, 10 : ops_10}
+        ops = {10 : ops_10}
```

Comment:
Good catch. Why did it fail before?

Reply:
check_onnx_opsets_operator() runs tests for opset versions in opset_versions, so it didn't run it for opset 9.